### ISSUE TITLE
Should we add a CODEOWNERS file?

### ISSUE THREAD

**Issue by techouse** [lejard-h/chopper#413]
Aloha contributors/maintainers, 😎

I was thinking that we should add a [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to automate assigning PR reviews.

Feel free to tag anyone appropriate so that we can build a list of active maintainers/contributors.

CC/ @lejard-h @JEuler @stewemetal @Guldem @meysam1717 @PixelToast

**Comment by JEuler** [comment]
Hmmm, I don't think that this is necessary, I think chopper is very close to its edge in terms of functionality, but of course, would be cool to have automation than to have manual work. :) 

We can discuss a kind of roadmap, I think. To know what we are going to add to the Chopper...

**Comment by Guldem** [comment]
Having a simple form of CODEOWNERS file shouldn't hurt I guess. Not sure how specific it should be. Did you @techouse already had something in mind? 

I can image that when having a lot of repositories to watch PR's could go unnoticed when not notified. I personally usually check some projects when I have the time so that wouldn't change much.

@JEuler having some kind of roadmap/vision for the project would indeed be nice. But I think that's sort of a different discussion. :)

**Comment by JEuler** [comment]
Yeah, it is different. Just related in terms of "what we can add to the Chopper" mind discussion.

**Comment by JEuler** [comment]
Hm, so are we going to add it? :)

**Comment by techouse** [comment]
Let's get a list together. :)

**Comment by JEuler** [comment]
Hm, I think everyone who is in this issue commenting is now around Chopper (for now) and can be added as contributor? :)

**Comment by meysam1717** [comment]
Hi guys.
I think it's could be helpful.

**Comment by techouse** [comment]
> Hm, I think everyone who is in this issue commenting is now around Chopper (for now) and can be added as contributor? :)

Right, created a draft PR https://github.com/lejard-h/chopper/pull/415
